### ISSUE TITLE
Download CHELSAcruts using ch_queries is not working

### ISSUE THREAD

**Issue by Jacksonmrodrigues** [matthewkling/chelsaDL#1]
Hi Matthew!

I am trying to download Tmin, Tmax and Prec from CHELSAcruts covering a period between 1961 - 2016.
However, your codes are not working properly for me. 
I would like to know if you can give some tips to make work.

#######
> ch_queries(variables = c("tmin","tmax", "prec"),timeframes = 1961:2016, 
+            months = 1:12) %>%
+   ch_dl()
Error in ch_queries(variables = c("tmin", "tmax", "prec"), timeframes = 1961:2016,  : 
  unused argument (months = 1:12)

######
Your example is not working also

> ch_queries(variables = c("temp", "prec"), models = "HadGEM2-ES", 
+            scenarios = c("rcp45", "rcp85"), timeframes = "2061-2080", 
+            months = 1:12) %>% 
+   ch_dl()
Error in ch_queries(variables = c("temp", "prec"), models = "HadGEM2-ES",  : 
  unused argument (months = 1:12)

Best wishes

Jackson

**Comment by matthewkling** [comment]
Hi Jackson, good catch, thanks! This error is because the example in the github readme was outdated -- the name of the parameter is now "layers" instead of "months". I've fixed the readme.

I also noticed as I was looking into this that the URL for the server where the chelsa data live has changed since I last updated the package, so I just fixed that too. If you install a fresh copy of the package, this should hopefully now work, but let me know if not:

```
ch_queries(variables = c("temp", "prec"), models = "ACCESS1-0", 
           scenarios = c("rcp45", "rcp85"), timeframes = "2061-2080", 
           layers = 1:12) %>%
    ch_dl()
```
Note also that `timeframes = 1961:2016` is not valid -- currently this package only supports downloading climatologies (not the time series data that has more recently been released), and as described in the documentation, the timeframes have to be one of the following options, as a character: "1979-2013", "2014-2060", or "2061-2080".
